File: sample/src/main/java/pl/droidsonroids/gif/sample/sources/GifSourcesFragment.java
package pl.droidsonroids.gif.sample.sources;

import android.content.Context;
import android.os.Bundle;
import android.support.v7.widget.LinearLayoutManager;
import android.support.v7.widget.RecyclerView;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import pl.droidsonroids.gif.sample.BaseFragment;

/**
 * Fragment with various GIF sources examples
 */
public class GifSourcesFragment extends BaseFragment {

    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
        final Context context = getContext();
        RecyclerView recyclerView = new RecyclerView(context);
        recyclerView.setLayoutManager(new LinearLayoutManager(context));
        recyclerView.setAdapter(new GifSourcesAdapter(context));
        return recyclerView;
    }
}

File: sample/src/main/java/pl/droidsonroids/gif/sample/sources/GifSourcesFragment.kt
package pl.droidsonroids.gif.sample.sources

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import androidx.fragment.app.Fragment
import androidx.recyclerview.widget.LinearLayoutManager
import androidx.recyclerview.widget.RecyclerView

/**
 * Fragment with various GIF sources examples
 */
class GifSourcesFragment : Fragment() {

	override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
		val recyclerView = RecyclerView(inflater.context)
		recyclerView.layoutManager = LinearLayoutManager(inflater.context)
		recyclerView.adapter = GifSourcesAdapter(inflater.context)
		return recyclerView
	}
}